android/src/main/java/org/jellyfin/apiclient/logging/AndroidLogger.kt

```diff
@@ -0,0 +1,30 @@
+package org.jellyfin.apiclient.logging
+
+import android.util.Log
+
+/**
+ * ILogger implementation for Android. Uses the [android.util.Log] class.
+ */
+class AndroidLogger(
+	private val tag: String = "jellyfin-apiclient"
+) : ILogger {
+	private fun getFormattedString(raw: String, vararg parameters: Any): String {
+		return String.format(raw, *parameters)
```

Comment:
I believe @AndreasGB had to make some changes to the initial logging class to avoid calling `String.format` when no parameters are present because it would cause an exception if the string contained a %. This doesn't seem to cover that case unless I am missing some Kotlin magic. :wink:

Reply:
As far as I could see all logging statements in the apiclient did not use string concatenation to add variables and didn't use the `%` character except for using parameters so it _should_ be safe without the check now as ATV moved to it's own logger too.

Reply:
I'd still program a bit defensively here to avoid shooting ourselves in the foot in the future.